Tests/test_Nexus.py

```diff
@@ -371,11 +727,17 @@ def test_NexusTest1(self):
 [T]    2.43     2.12     2.31      .       2.14     
 [-]    2.43     2.14     2.31     2.14      .       
 ;
-""")  # noqa : W291
+""",
+        )  # noqa : W291
```

Comment:
You will need to manually move the comment ``# noqa : W291`` up one line to be where the multi-line string ends